changelogs/54140.yml

```
@@ -0,0 +1,4 @@
+title: "delete a not used field"
+type: fix
+description: |
+  We remove the unused field "lead" from EmailAddress.
```

```suggestion
title: "Remove the unused field 'lead' from email address"
type: fix
```

It's better to keep it simple, description is not mandatory :-)